test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -181,19 +181,19 @@ CREATE INDEX IF NOT EXISTS tx_status_by_hash_hash_index ON tx_status_by_hash USI
 
 CREATE TABLE IF NOT EXISTS height_by_account_set (
     account_id text,
-    height text
+    height bigint
 );
 CREATE TABLE IF NOT EXISTS index_by_creator_height (
     id serial,
     creator_id text,
-    height text,
-    index text
+    height bigint,
+    index bigint
 );
 CREATE TABLE IF NOT EXISTS index_by_id_height_asset (
```

Comment:
Can you check why is `index_by_id_height_asset` table is missing in `StorageImpl`, and `position_by_account_asset` is missing here?
https://github.com/hyperledger/iroha/blob/375512635e5619c0346c01ea61c4c902ce1c2639/irohad/ametsuchi/impl/storage_impl.cpp#L990
I think this discrepancy has to be resolved before merging this pull request.

Reply:
Left a todo